api/database/helpers/ApiErrorEnums.php

```diff
@@ -25,7 +25,18 @@ class ApiErrorEnums
 
     const PROCESS_CLOSING_DATE_EXTEND = 'UpdatePoolClosingDateExtend';
 
+    // pool candidate field validation
+    const EXPIRY_DATE_REQUIRED = 'ExpiryDateRequired';
```

Comment:
I elected not to do that for a few reasons, let me know what you think
- I wasn't sure copy changes were in scope
- Whether error message copy was decided yet
- There are other error messages too and to group them in one issue to avoid missing any, such as the above errors as well as the ones for revert final decision and the placing pair of errors. So, ensuring all error codes in ErrorEnums have copy in one ticket
- To not block potentially for a while https://github.com/GCTC-NTGC/gc-digital-talent/issues/9197

Reply:
Yeah, those are all good reasons.  Could you create that issue and make sure it's part of the ROD2 milestone?

Reply:
Issue https://github.com/GCTC-NTGC/gc-digital-talent/issues/10075